internal/pkg/vm/internal/vmcontext/vmcontext.go

```diff
@@ -396,16 +395,24 @@ func (vm *VM) applyMessage(msg *types.UnsignedMessage, onChainMsgSize int, rnd c
 
 	// 5. Increment sender CallSeqNum
 	fromActor.IncrementSeqNum()
+	// update actor
```

Comment:
@icorderi out of band you mentioned there was a way to do the same fix by only moving two lines.  Please point it out in the comments here if that's still the case.

Reply:
Disregard I see it here https://github.com/filecoin-project/go-filecoin/pull/3857/files

Reply:
I have this fixes and a few others waiting to merge in https://github.com/filecoin-project/go-filecoin/pull/3857

I would rebase this PR as soon as that one merges.